boards/arm/nrf5340pdk_nrf5340/nrf5340pdk_nrf5340_cpuapp_common.dts

```diff
@@ -138,15 +138,15 @@
 
 		boot_partition: partition@0 {
 			label = "mcuboot";
-			reg = <0x00000000 0x10000>;
+			reg = <0x00000000 0x0000C000>;
 		};
-		slot0_partition: partition@10000 {
+		slot0_partition: partition@c000 {
```

Comment:
Are changes to nrf5340pdk_nrf5340_cpuapp_partition_conf.dts also required? Here and elsewhere in this PR. Also nrf9160_innblue21_partition_conf.dts, etc.

Reply:
Sorry missed them.